CONTRIBUTING.md

```diff
@@ -188,9 +188,11 @@ You are almost ready to make changes to files, but before that you should **alwa
 
 5. Test your code **Always!** 
 
+    * Before starting the application confirm that `npm run lint` and then `npm run test` pass.  Fix all errors that do not pass.
```

Comment:
Would it be advisable to have a script command in package.json that runs all the necessary tests at once?  

@ojeytonwilliams is there a common pattern with running all tests at once, like
_npm run all-tests_
vs having people individually run each test?  

For instance, the _"both"_ script has an example of concurrently running multiple commands with a single script command. 

`    "both": "concurrently \"npm run dev\" \"cd client && npm run dev\"",
`

Also, since the two bullet point directly below this line are specific to Docker Mode and Manual Mode, how about we remove this new line and update those two bullets as needed?

Reply:
Sure, we could do that.  I don't know that it's a common pattern, but it seems perfectly reasonable to me.  I'd call it `lint-and-test`, though, just because `test` is implicitly "run all tests".  At least that's how I interpret it.

> Also, since the two bullet point directly below this line are specific to Docker Mode and Manual Mode, how about we remove this new line and update those two bullets as needed?

Sounds good.